packages/ui/aurora/src/components/Selector/Selector.tsx

```diff
@@ -0,0 +1,205 @@
+
+  return (
+    <PopperPrimitive.Anchor asChild {...popperScope}>
+      <div className='flex items-center gap-1'>
+        <Input.Root>
+          <Input.TextInput
+            {...getInputProps()}
+            placeholder={placeholder}
+            variant='subdued'
+            classNames={tx('selector.input', 'selector__input', {}, classNames)}
+          />
+        </Input.Root>
+        <Button
+          {...getToggleButtonProps()}
+          aria-label='toggle menu'
+          variant='ghost'
+          classNames={tx('selector.button', 'selector__button', {}, classNames)}
+        >
+          {/* TODO(burdon): SelectPrimitive.Icon? */}
+          {(isOpen && <CaretUp />) || <CaretDown />}
+        </Button>
+      </div>
+    </PopperPrimitive.Anchor>
+  );
+};
+
+//
+// Content
+//
+
+const CONTENT_NAME = 'SelectorContent';
+
+type SelectorContentProps = ThemedClassName<ScopedProps<PropsWithChildren<{}>>>;
+
+const SelectorContent = forwardRef<HTMLDivElement>(
+  ({ __scopeSelector, classNames }: SelectorContentProps, forwardedRef) => {
+    const { tx } = useThemeContext();
+    const popperScope = usePopperScope(__scopeSelector);
+    const { adapter, items, getMenuProps, getItemProps, highlightedIndex, selectedItem, isOpen } = useSelectorContext(
+      CONTENT_NAME,
+      __scopeSelector,
+    );
+
+    if (!isOpen) {
+      return null;
+    }
+
+    return (
+      <div>
+        <PopperPrimitive.Content
+          data-state={isOpen}
+          role='dialog'
+          {...popperScope}
+          ref={forwardedRef}
+          style={
+            {
+              // re-namespace exposed content custom properties
+              ...{
+                '--radix-selector-content-transform-origin': 'var(--radix-popper-transform-origin)',
+                '--radix-selector-content-available-width': 'var(--radix-popper-available-width)',
+                '--radix-selector-content-available-height': 'var(--radix-popper-available-height)',
+                '--radix-selector-trigger-width': 'var(--radix-popper-anchor-width)',
+                '--radix-selector-trigger-height': 'var(--radix-popper-anchor-height)',
+              },
+            } as any // TODO(burdon): Why is this needed?
+          }
+        >
+          <ul {...getMenuProps()} className={tx('selector.content', 'selector__content', {}, classNames)}>
+            {items.map((item, index) => (
+              <li
+                key={adapter(item).id}
+                data-selected={selectedItem === item ? 'true' : undefined}
+                data-highlighted={highlightedIndex === index ? 'true' : undefined}
+                {...getItemProps({
+                  index,
+                  item,
+                  className: tx('selector.item', 'selector__item', {}, classNames),
+                })}
+              >
+                {adapter(item).text}
+              </li>
+            ))}
+          </ul>
+        </PopperPrimitive.Content>
+      </div>
```

Comment:
This will need to be refactored to export subcomponents to match ontologies with the rest of aurora.